server/seeders/seeds.js

```diff
@@ -1,120 +1,89 @@
-        { _id: userId },
-        { $push: { savedMovies: { movieId, release, overview, image, vote, voteCount, name, username } } },
-        { runValidators: true }
-      );
-    }
-
-// create removedMovies
-for (let i = 0; i < 100; i += 1) {
-  const movieId = faker.address.zipCode();
-  const release = faker.lorem.word();
-  const overview = faker.lorem.words(Math.round(Math.random() * 20) + 1);
-  const image = faker.image.imageUrl();
-  //example//'https://image.tmdb.org/t/p/w185_and_h278_bestv2/rplLJ2hPcOQmkFhTqUte0MkEaO2.jpg'
-  const vote = faker.random.number()*5;
-  const voteCount = faker.random.number()*5;
-  const name = faker.commerce.productName();
-  //example//"The Silence of the Lambs"
-  const randomUserIndex = Math.floor(Math.random() * createdUsers.ops.length);
-  const { username } = createdUsers.ops[randomUserIndex];
-  const { _id: userId } = createdUsers.ops[randomUserIndex];
-
-  await User.updateOne(
-    { _id: userId },
-    { $push: { removedMovies: { movieId, release, overview, image, vote, voteCount, name, username } } },
-    { runValidators: true }
-  );
-}
-
   // create friends
   for (let i = 0; i < 100; i += 1) {
     const randomUserIndex = Math.floor(Math.random() * createdUsers.ops.length);
     const { _id: userId } = createdUsers.ops[randomUserIndex];
-
     let friendId = userId;
-
     while (friendId === userId) {
       const randomUserIndex = Math.floor(Math.random() * createdUsers.ops.length);
       friendId = createdUsers.ops[randomUserIndex];
     }
-
     await User.updateOne({ _id: userId }, { $addToSet: { friends: friendId } });
   }
 
-  // create Comments
-  let createdComments = [];
+  // create likedMovies
+  let likedMovies = [];
   for (let i = 0; i < 100; i += 1) {
-    const commentText = faker.lorem.words(Math.round(Math.random() * 20) + 1);
+    const externalMovieId = faker.random.number();
+    const rating = faker.finance.amount();
+    //(min?: 1, max?: 10)
+    const voteCount = faker.random.number();
+    const title = faker.commerce.productName();
+    const overview = faker.lorem.words(Math.round(Math.random() * 20) + 1);
+    const releaseDate = faker.date.past();
+    const poster = faker.image.imageUrl();
+    const trailer = faker.image.imageUrl();
+    //example//'https://image.tmdb.org/t/p/w185_and_h278_bestv2/rplLJ2hPcOQmkFhTqUte0MkEaO2.jpg'
+//externalMovieId, rating, voteCount, title, overview, releaseDate, poster, trailer
 
     const randomUserIndex = Math.floor(Math.random() * createdUsers.ops.length);
     const { username, _id: userId } = createdUsers.ops[randomUserIndex];
 
-    const createdComment = await Comment.create({ commentText, username });
+    const likedMovie = await Movie.create({ externalMovieId, rating, voteCount, title, overview, releaseDate, poster, trailer, username });
 
     const updatedUser = await User.updateOne(
       { _id: userId },
-      { $push: { comments: createdComment._id } }
+      { $push: { movies: likedMovie._id } }
```

Comment:
should this be `{ $addToSet: { likedMovies: likedMovie._id } }`?

Reply:
not quite, but I have refactored the code differently so you have likedMovies and dislikedMovies